stage0/run.go

```diff
@@ -258,9 +258,11 @@ func Run(cfg RunConfig, dir string) {
 
 	if cfg.SpawnMetadataService {
 		log.Print("Launching metadata svc")
-		if err := launchMetadataService(cfg.Debug); err != nil {
-			log.Printf("Failed to launch metadata svc: %v", err)
-		}
+		go func() {
+			if err := launchMetadataService(cfg.Debug); err != nil {
```

Comment:
@eyakubovich why isn't this a critical error? maybe we could use a comment to clarify this.